core-feature-pack/src/main/resources/configuration/domain/template.xml

```diff
@@ -77,6 +77,9 @@
         <server-group name="main-server-group" profile="default">
             <jvm name="default">
                 <heap size="64m" max-size="512m"/>
+                <environment-variables>
+                    <variable name="MALLOC_ARENA_MAX" value="1"/>
+                </environment-variables>
```

Comment:
This should go in the host.xml, host-master.xml and host-slave.xml templates rather than here. Here it only applies to servers in this group. The expectation is the jvm setting in server-group is just to override some settings from host.xml. Having one in our standard domain.xml is really just an illustration.

Leaving it here too is ok as a second line of defense if you want.

Note that this change doesn't satisfy any requirement in the JBEAP as the templates in full-feature-pack are the ones that are relevant to the EAP dist and the full WildFly dist. So changes in full are needed too.